federation-1/harmonizer/esbuild/bundler.js

```diff
@@ -1,13 +1,33 @@
+let denoFsPlugin = {
+  name: "fs",
+  setup(build) {
+    // Intercept require("fs") and replace it with shims
+    build.onResolve({ filter: /^fs$/ }, (args) => ({
+      path: args.path,
+      namespace: "deno-fs",
+    }));
+
+    build.onLoad({ filter: /.*/, namespace: "deno-fs" }, () => ({
+      contents:
+        "module.exports = { existsSync: Deno.ensureFileSync, writeFileSync: Deno.writeTextFile }",
+      loader: "js",
+    }));
```

Comment:
do we need this for sure? i'm hesitant to add shims for the file system when really the upstream libraries shouldn't depend on these APIs being present. they're just running on inputs/outputs to a function and shouldn't have any side effects like writing to the file system - rover or other library consumers would take care of that bit

Reply:
You're correct, we don't need this. I pushed a change